### ISSUE TITLE
Joined Groupchats do not appear in UI

### ISSUE THREAD

**Issue by haroldkyle** [igniterealtime/pade#217]
I think that when I join a Groupchat, the chat room is supposed to appear under "Groupchats" in the UI. Nothing appears there when I'm in a room. When I've joined several rooms, and a notification for a new message happens, there's no way to tell which room prompted the notification. No list of joined groupchats appears in the interface at all, that I can find at the moment.

![groupchats](https://user-images.githubusercontent.com/2402898/78682413-ed14c400-78bb-11ea-8473-e73b0ac435fe.png)

**Comment by deleolajide** [comment]
It should take you to the  converse registration/login screen, but that is currently broken.

Check openfire. You are probably logged in as anonymous user. Go to https://your_server:7443/pade/options.html and set your username and password.

**Comment by deleolajide** [comment]
Could you try using the bookmarks plugin for Openfire to create a few bookmarks and confirm if they show.

**Comment by haroldkyle** [comment]
Funny you should bring up Bookmarks. These are also failing. From the console:
```
<error code="503" type="cancel"><service-unavailable xmlns="urn:ietf:params:xml:ns:xmpp-stanzas"/></error>
```
I hadn't mentioned the bookmark error because my plugins were out of date and it seemed like a server issue. I have now updated all plugins (Bookmarks to 1.0.3) and also Openfire itself to 4.5.1. I still cannot see open groupchats nor save a bookmark. Also, a bookmark I created on the server directly is not properly fetched at launch. The error is the same 503 as above.

**Comment by deleolajide** [comment]
There are 2 types of bookmarks. Bookmarks created by the client that are stored server-side as PEP data and bookmarks created on the server by the admin. Looks like none are working.  

The only time I have seen this happen is when two different clients like Conversations and Converse try to use the same bookmarks and  they stop working. All I can suggest is delete all PEP bookmarks, do a factory reset in Pade and start again.

![image](https://user-images.githubusercontent.com/110731/78801025-0b5deb00-79b4-11ea-9b20-7ec3fd9dc8e7.png)

**Comment by haroldkyle** [comment]
I don't have a section for Users > User Options. Here is what my installation looks like:
![users](https://user-images.githubusercontent.com/2402898/78818035-06903b80-79a2-11ea-817a-47fdeb9927c0.png)

Separately, I see bookmarks (empty) under Group Chats:
![bookmarks](https://user-images.githubusercontent.com/2402898/78818058-0c861c80-79a2-11ea-8470-ca25faf66a1c.png)

I made sure to perform a factory reset Pade prior to posting this bug. I factory reset again after updating plugins earlier today. Still, no change. Is there anything else I could do to help track this bug down?

**Comment by haroldkyle** [comment]
Ah, when I click on a user on the server, I see "Personal Eventing" tab that's throwing an Exception. I'll track that down.

**Comment by haroldkyle** [comment]
OK I found the "Personal Eventing" under my individual user. I deleted all leaf nodes, including one for bookmarks. I performed another factory reset. Still no luck.

**Comment by deleolajide** [comment]
I finally found the issue. It is fixed in 1.6.3
